cmake/VWFlags.cmake

```diff
@@ -18,9 +18,14 @@ if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
   endif()
 endif()
 
+set(LINUX_ARM64_OPT_FLAGS "")
+if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "aarch64|arm64|ARM64")
+  set(LINUX_ARM64_OPT_FLAGS -mcpu=neoverse-n1)
+endif()
```

Comment:
This is specific to this CPU, so we should not be adding it whenever we encounter an arm CPU. Generally for specific optimization flags like this it is better to add them when configuring your build.

So, in this case we should remove this but in your build you would add the following to your command line to achieve the same outcome:
```
-DCMAKE_CXX_FLAGS_RELEASE="-mcpu=neoverse-n1"
```

Reply:
I'm going to go ahead and push a change removing this piece so we can move forward here.

Reply:
@jackgerrits FYI, I added `sse2neon` to vcpkg. Would you prefer that I add the package instead?